### PR TITLE
Create ricky.json

### PR DESCRIPTION
**Pull request by rickypurogu**

### CODE REVIEW

File: sub-logs/ricky.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "email": "tanas.riccardo03@gmail.com"
+  },
+
+  "target": {
+    "CNAME": {
+            "name": "ricky",
+            "value": "rickypurogu.vercel.app"
+      },
+    "TXT": {
+       "name": "_vercel",
```
Comment (DIBSTERYT) [inline sub-logs/ricky.json:15]: You cannot have a CNAME record with other records present. CNAME must exist by itself without anything else.

https://www.rfc-editor.org/rfc/rfc1034#section-3.6.2

**Review by DIBSTERYT — CHANGES_REQUESTED** [review]

File: sub-logs/ricky.json (hunk 1 above)
Comment (rickypurogu) [reply sub-logs/ricky.json:11]: My bad, I thought it was necessary.

**Comment by DIBSTERYT** [comment]
Little confused here, it appears you want just a TXT record? TXT would just verify you own it, but it wouldn't point the domain anywhere, which I assume is what you would want to do.

**Comment by rickypurogu** [comment]
Vercel isn't providing me with much info on how to completely set up a domain, so I structured the file on what others did. Everyone who is using Vercel has both a CNAME record and the TXT record needed by Vercel for the domain verification.
I suppose I have to add the CNAME record for it to work, am I right?

**Comment by WilliamDavidHarrison** [comment]
> Vercel isn't providing me with much info on how to completely set up a domain, so I structured the file on what others did. Everyone who is using Vercel has both a CNAME record and the TXT record needed by Vercel for the domain verification.
> I suppose I have to add the CNAME record for it to work, am I right?

Use the `A` record `76.76.21.21` instead of the CNAME and it will work.

**Review by WilliamDavidHarrison — APPROVED** [review]

**Comment by rickypurogu** [comment]
> Use the `A` record `76.76.21.21` instead of the CNAME and it will work.
Thanks, still no clue why others have both CNAME and TXT with Vercel. Did something change lately?

**Comment by Tweak4141** [comment]
Check your json, something isn't lining up. Check failing, will retrigger on branch push.

**Review by DIBSTERYT — APPROVED** [review]

**Review by WilliamDavidHarrison — APPROVED** [review]

**Pull request merged**